framework/src/modules/chain/transaction_pool/transaction_pool.js

```diff
@@ -89,22 +90,28 @@ class TransactionPool extends EventEmitter {
 		this.validateTransactions = transactionsModule.validateTransactions(
 			this.exceptions,
 		);
-		this.verifyTransactions = transactionsModule.composeTransactionSteps(
-			transactionsModule.checkAllowedTransactions(() => {
-				const { version, height, timestamp } = this.blocks.lastBlock;
-				return {
-					blockVersion: version,
-					blockHeight: height,
-					blockTimestamp: timestamp,
-				};
-			}), // TODO: probably wrong
-			transactionsModule.checkPersistedTransactions(storage),
-			transactionsModule.verifyTransactions(storage, slots, exceptions),
-		);
-		this.processTransactions = transactionsModule.composeTransactionSteps(
-			transactionsModule.checkPersistedTransactions(storage),
-			transactionsModule.applyTransactions(storage, exceptions),
-		);
+		this.verifyTransactions = transactions => {
+			const stateStore = new StateStore(this.storage);
+			return transactionsModule.composeTransactionSteps(
+				transactionsModule.checkAllowedTransactions(() => {
+					const { version, height, timestamp } = this.blocks.lastBlock;
+					return {
+						blockVersion: version,
+						blockHeight: height,
+						blockTimestamp: timestamp,
+					};
+				}), // TODO: probably wrong
```

Comment:
I know this comment was not introduced in this PR but this is a very weird message 😄 

This should be investigated or removed IMO